src/mongocxx/collection.hpp

```diff
@@ -624,7 +624,7 @@ MONGOCXX_INLINE bsoncxx::stdx::optional<result::insert_many> collection::insert_
     document_view_iterator_type end,
     const options::insert& options
 ) {
-    class bulk_write writes(false);
+    class bulk_write writes(options.ordered().value_or(true));
```

Comment:
That was intentional! The crud api specifies bulk ops defaulting to ordered
[(specified in BulkWriteOptions)](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst)